pkg/image/image.go

```diff
@@ -186,6 +190,40 @@ func Pull(location string, ref *Reference, auth *k8s.AuthConfig) (img *Info, err
 	}, nil
 }
 
+// LibraryInfo queries remote library to get info about the image.
+// If image is not found returns ErrNotFound. For references other than
+// library returns nil, nil.
```

Comment:
but if user sends a references other than library it have to be an error, isn't it?

Reply:
returning nil, nil it's not a really good practice

Reply:
but why you returning nil? users provides an invalid input to the function. It's an error

Reply:
I would better return ErrNotLibraryDomain or similar

Reply:
input is an arbitrary reference and docker reference is still valid
the main idea in returning nil,nil is to say "hey, no library image corresponds to your reference"